src/Badge.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import { Box, IBox } from './Box';
+
+export enum BadgeVariant {
+  Pill = 'pill',
+  Textual = 'textual',
+}
+
+export enum BadgeColor {
+  Error = 'error',
+  Warning = 'warning',
+  Default = 'default',
+}
+
+const colorMap = {
+  [BadgeColor.Error]: '#dc3546',
+  [BadgeColor.Warning]: '#ffc108',
+  [BadgeColor.Default]: '#6c757e',
+};
+
+export interface IBadge extends IBox<HTMLElement> {
+  label: string;
+  fixedWidth?: boolean;
+  inverse?: boolean;
+  listItem?: boolean;
+  variant?: BadgeVariant;
+  color?: BadgeColor;
+}
+
+export const Badge: React.FunctionComponent<IBadge> = props => {
+  const { label, color = BadgeColor.Error, variant = BadgeVariant.Pill, ...restProps } = props;
+
+  return (
+    <Box {...restProps} as="span" css={badgeStyles({ color, variant })}>
+      {label}
```

Comment:
I'd say we should go with children here so that the following usage is allowed:
```jsx
<Badge>2</Badge>
```

Reply:
👍